DeskReserve/Repository/BuildingRepository.cs

```diff
@@ -22,13 +21,13 @@ public async Task<List<Building>> GetAllAsync()
 
         public async Task<Building> GetByIdAsync(Guid id)
         {
-            return await _dbContext.Buildings.FindAsync(id);
+            return await _dbContext.Buildings.FindAsync(id) ?? throw new DataNotFound("Data don't exist");
```

Comment:
Just a recommendation for next time create class exception messages constatants and use them